go.mod

```diff
@@ -1,8 +1,16 @@
-module github.com/abrander/garmin-connect
+module github.com/garmin-connect
```

Comment:
This looks wrong.

Reply:
Ya that is weird i dont know why go did that when i refreshed the go.mod for 1.20

Reply:
Figured it out . It was due to my folder structure